packages/playwright-test/src/abq/index.ts

```diff
@@ -80,7 +84,11 @@ function applyAbqConfiguration(config: FullConfig) {
     console.warn(`Warning: ABQ does not support Playwright sharding. Overriding configuration value of '${JSON.stringify(config.shard)}'.`);
     config.shard = null;
   }
+
+  // Disable built-in reporters to use ABQ's reporter.
+  config.reporter = [['null', undefined]];
```

Comment:
What's the implication if customers have written custom reporters for anything? Are they expected to consume RWX v1 JSON instead?

Reply:
Yes, I'm documenting this in an upcoming rwx.com PR.